packages/flutter/lib/src/cassowary/solver.dart

```diff
@@ -231,17 +239,23 @@ class Solver {
     return _optimizeObjectiveRow(_objective);
   }
 
+  /// Check if the given constraint is present in the solver.
   bool hasConstraint(Constraint constraint) {
     return _constraints.containsKey(constraint);
   }
 
+  /// Add a list of edit variable to the solver at a given priority. Either
```

Comment:
"Adds".

I don't know what "No edit variables may be added at the required priority." means.